apps/cloudappclient/app.js

```diff
@@ -119,19 +119,20 @@ module.exports = activity => {
     var playerId
     logger.log(`exe dt: media.${dt.action}`)
     function setSpeed (speed) {
-      if (typeof speed === 'number') {
-        activity.media.setSpeed(speed, pm.getByAppId(dt.data.appId))
-      }
+      return activity.media.setSpeed(speed, pm.getByAppId(dt.data.appId))
     }
     function setOffset (offset) {
-      if (typeof offset === 'number' && offset >= 0) {
-        activity.media.seek(offset, pm.getByAppId(dt.data.appId))
-      }
+      activity.media.seek(offset, pm.getByAppId(dt.data.appId))
```

Comment:
Should the promise be returned here?

Reply:
No need. But it is better to catch unexpected errors.